### PR TITLE
CTUSA dns

### PR DESCRIPTION
**Pull request by rahearn**
not to be merged until later

### CODE REVIEW

File: terraform/covidtest.usa.gov.tf [hunk 1]
```diff
@@ -79,458 +79,278 @@ resource "aws_route53_record" "acme_challenge_eastb_covidtest_usa_gov_cname" {
...
+  type           = "AAAA"
+  set_identifier = "eb"
+  weighted_routing_policy {
+    weight = 10
+  }
+  health_check_id = aws_route53_health_check.eastb_covidtest_usa_gov_health.id
+  alias {
+    name                   = "eastb-domains-0-930753903.us-gov-east-1.elb.amazonaws.com"
+    zone_id                = "Z166TLBEWOO7G0"
+    evaluate_target_health = false
```
Comment (bengerman13) [inline terraform/covidtest.usa.gov.tf:354]: do we want true for these?

Comment (bengerman13) [reply terraform/covidtest.usa.gov.tf:354]: I think I see now, nevermind

**Review by bengerman13 — APPROVED** [review]
LGTM

We talked about switching to favicon for health check - I'm assuming we deliberately did't make that switch

**Comment by rahearn** [comment]
> I'm assuming we deliberately did't make that switch

correct, the favicon also has a hash digest so didn't want to risk unhealthy because it changed. `robots.txt` is small, consistent, and served by a very early piece of middleware so should be low impact on the app overall

**Comment by bengerman13** [comment]
> robots.txt is small, consistent, and served by a very early piece of middleware so should be low impact on the app overall

that sounds like we're switching to `robots.txt` , but what I see still says `/`

**Comment by rahearn** [comment]
> that sounds like we're switching to `robots.txt` , but what I see still says `/`

yeah - switched to robots.txt for staging, will switch production after verifying that the robots version works properly. We have a baseline understanding of how this one operates

File: terraform/covidtest.usa.gov.tf [hunk 2]
```diff
@@ -79,458 +79,278 @@ resource "aws_route53_record" "acme_challenge_eastb_covidtest_usa_gov_cname" {
...
+  port              = 443
+  type              = "HTTPS_STR_MATCH"
+  resource_path     = "/"
+  search_string     = "COVIDtests.gov"
+  failure_threshold = "2"
+  request_interval  = "30"
+}
+
+# route -> westb
+resource "aws_route53_record" "route_covidtest_usa_gov_a_wb" {
```
Comment (bengerman13) [inline terraform/covidtest.usa.gov.tf:138]: we should set TTL here to 5 seconds per AWS suggestion

Comment (rahearn) [reply terraform/covidtest.usa.gov.tf:138]: do you have a doc link? I thought that you couldn't set TTL on an alias

Comment (bengerman13) [reply terraform/covidtest.usa.gov.tf:138]: https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/resource-record-sets-values-multivalue.html#rrsets-values-multivalue-ttl

this is what makes me think you can. I'm testing in the cloud.gov account right now

Comment (bengerman13) [reply terraform/covidtest.usa.gov.tf:138]: further googling shows this isn't possible

**Review by LindsayYoung — APPROVED** [review]
LATO signing completed on 1/17/2022 02:44:13 pm est

**Pull request merged**